hack/jenkins/e2e.sh

```diff
@@ -555,6 +555,8 @@ case ${JOB_NAME} in
     NUM_NODES="100"
     # Reduce logs verbosity
     TEST_CLUSTER_LOG_LEVEL="--v=2"
+    # TODO: Remove when we figure out the reason for ocassional failures #19048
+    KUBELET_TEST_LOG_LEVEL="--v=4"
```

Comment:
What I meant is that I though that all logs that can give us some value (like pod received from watch) are at verbosity 2. Or am I missing something?